Assets/Scripts/Models/Furniture.cs

```diff
@@ -143,15 +146,25 @@ private Furniture(Furniture other)
         UnlocalizedDescription = other.UnlocalizedDescription;
     }
 
+    /// <summary>
+    /// This event will trigger when the furniture has been changed.
+    /// </summary>
     public event Action<Furniture> Changed;
 
+    /// <summary>
+    /// This event will trigger when the furniture has been removed.
+    /// </summary>
     public event Action<Furniture> Removed;
 
+    /// <summary>
+    /// This event will trigger if <see cref="IsOperating"/> has been changed.
+    /// </summary>
     public event Action<Furniture> IsOperatingChanged;
 
     /// <summary>
-    /// Gets or sets the Furniture's pathfinding modifier which is added into the Tile's final PathfindingCost.
+    /// Gets or sets the Furniture's <see cref="PathfindingModifier"/> which is added into the Tile's final PathfindingCost.
```

Comment:
It could be:

>  Gets or sets the Furniture's pathfinding modifier which is added into the final PathfindingCost computation

Just remove the Tile reference.